proxy/src/main/java/org/apache/rocketmq/proxy/processor/ReceiptHandleProcessor.java

```diff
@@ -192,7 +193,7 @@ protected CompletableFuture<MessageReceiptHandle> startRenewMessage(MessageRecei
                         messageReceiptHandle.resetRenewRetryTimes();
                         resFuture.complete(messageReceiptHandle);
                     } else {
-                        log.error("renew response is not ok. result:{}, handle:{}", ackResult, messageReceiptHandle, throwable);
+                        log.error("renew response is not ok. result:{}", ackResult, messageReceiptHandle);
```

Comment:
The format is not right, a `{}` is missing here

Reply:
The throwable argument is missing for log.error

Reply:
> The throwable argument is missing for log.error

In this judgment condition, the throwable is always empty, so no record is required.